src/DotNetWorker.Grpc/Handlers/InvocationHandler.cs

```diff
@@ -40,7 +40,7 @@ public InvocationHandler(
             _invocationFeaturesFactory = invocationFeaturesFactory ?? throw new ArgumentNullException(nameof(invocationFeaturesFactory));
             _outputBindingsInfoProvider = outputBindingsInfoProvider ?? throw new ArgumentNullException(nameof(outputBindingsInfoProvider));
             _inputConversionFeatureProvider = inputConversionFeatureProvider ?? throw new ArgumentNullException(nameof(inputConversionFeatureProvider));
-            _workerOptions = workerOptions?.Value ?? throw new ArgumentNullException(nameof(workerOptions));
+            _workerOptions = workerOptions.Value ?? throw new ArgumentNullException(nameof(workerOptions));
```

Comment:
Should we make `_inflightInvocations` readonly on line 29?

Reply:
Done